packages/framework/undo-redo/src/test/matrix.spec.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License.
  */
 
+// eslint-disable-next-line unicorn/prefer-node-protocol
```

Comment:
Instead of disabling, update the import to be `from "node:assert"`. We do it explicitly for anything that comes from `node`.

Reply:
I found an overwritten rule in [eslintrc.cjs](https://github.com/microsoft/FluidFramework/pull/24382/files#diff-3977ca47573a55cadb62426a97db265f395fab948c9b0015d7992c6d4c715517) speciafically allowing "assert" here, so I think I better to keep it.

Reply:
That rule is for a slightly different thing; it allows us to even try to import from Node's `assert` module (instead of complaining that we shouldn't use it). But now that the rule says we can use it, a separate rule is asking that we specify it as `node:assert` instead of just `assert`. I still think we should update it.